markups/asciidoc.py

```diff
@@ -0,0 +1,99 @@
+# vim: ts=4:sts=4:sw=4:expandtab
+
+# This file is part of python-markups module
+# License: 3-clause BSD, see LICENSE file
+# Copyright: (C) Dmitry Shachnev, 2012-2021
```

Comment:
Do you want to put yourself in Copyright here and in the test?

And year can be just 2022.